airflow/bin/cli.py

```diff
@@ -456,14 +448,12 @@ def _run(args, dag, ti):
         if args.ship_dag:
             try:
                 # Running remotely, so pickling the DAG
-                session = settings.Session()
-                pickle = DagPickle(dag)
-                session.add(pickle)
-                session.commit()
-                pickle_id = pickle.id
-                # TODO: This should be written to a log
-                print('Pickled dag {dag} as pickle_id:{pickle_id}'
```

Comment:
Such formatting of messages causes that a string object is created, which then may not be used anywhere, when the login level will be too low.
Formatting parameters should be passed as arguments to the `info` method.
Example:

````python
self.log.info("The Table '%s' does not exists already.", self.table_id)
````

Reply:
Good point @mik-laj 

The line is unrelated to the code, but I will push the fix in this PR anyway.